### PR TITLE
Fix #589 Reaction emoji no longer misaligns in android

### LINKED ISSUE #589: Sometimes numbers in emoji reactions are not aligned correctly
This can be observed both on Android and iOS but not always.

### PR DESCRIPTION
**Pull request by nashvail**
Had to include one line of platform specific code because of the way android and ios handle nested element heights differently.

### CODE REVIEW

File: src/reactions/Reaction.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ import Emoji from '../emoji/Emoji';
 import { getAuth } from '../account/accountSelectors';
 import { emojiReactionAdd, emojiReactionRemove } from '../api';
 
+const REACTION_HEIGHT = 30;
+const SPINNER_OFFSET = (Platform.OS === 'ios') ? REACTION_HEIGHT : REACTION_HEIGHT - 2;
```
Comment (vishwesh3) [inline src/reactions/Reaction.js]: Can we do this through https://github.com/zulip/zulip-mobile/blob/master/src/styles/platform.android.js and https://github.com/zulip/zulip-mobile/blob/master/src/styles/platform.ios.js

Comment (nashvail) [inline src/reactions/Reaction.js]: yeah! thanks @vishwesh3 wasn't aware of this, I'll give it a shot.

Comment (borisyankov) [inline src/reactions/Reaction.js]: 👍

**Comment by borisyankov** [comment]
Did you find out why there is a difference between Android and iOS sizing?

**Comment by nashvail** [comment]
@borisyankov on setting the height of a single number in spinner equal to the Reaction container's height, on iOS the border around the container wasn't included in the height while on Android it was. That's why the -2 in (REACTION_SPINNER_OFFSET = REACTION_HEIGHT - 2)

**Comment by borisyankov** [comment]
Good to know. Any chance this is a bug in React Native? Maybe report it?

**Comment by nashvail** [comment]
Maybe yes ... I'll dig further into this and open an issue

**Pull request merged**